api/src/changeRequests/validateChangeRequestAccess.ts

```diff
@@ -37,10 +37,25 @@ export async function validateChangeRequestAccess(
         // -----------------------
         const editGroups = accessMap.calculateAccess([DocType.Group], AclPermission.Edit);
         if (!editGroups.includes(doc._id)) {
-            return {
-                validated: false,
-                error: "No access to 'Edit' document type 'Group'",
-            };
+            // If this is a new group, the permission system does not yet know about it and it will not be in the access map.
+            // We need to check if the user has edit access to at least one of the groups in the ACL list, which will imply that
+            // the user has edit access to the new group through inheritance.
+            const aclEditGroups = accessMap.calculateAccess([DocType.Group], AclPermission.Edit);
```

Comment:
Is it needed to calculate this again? This variable seems to be the same as `editGroups` above.

Reply:
Silly of me yes